main.go

```diff
@@ -126,13 +126,11 @@ type options struct {
 }
 
 func main() {
-	// configure glog
-	flag.CommandLine.Parse([]string{})
-	flag.Lookup("logtostderr").Value.Set("true")
-
 	options := &options{collectors: make(collectorSet)}
 	flags := pflag.NewFlagSet("", pflag.ExitOnError)
-
+	// add glog flags
+	flags.AddGoFlagSet(flag.CommandLine)
+	flags.Lookup("logtostderr").Value.Set("true")
```

Comment:
@drinktee @brancz 

Currently, `flags.Lookup("logtostderr").Value.Set("true")` will set `logtostderr` to `true` which will be conflict with the help output for `logtostderr`.

By adding  
```
flags.Lookup("logtostderr").DefValue = "true"
flags.Lookup("logtostderr").NoOptDefVal = "true"
```

On this way we can also set the default value for `logtostderr` to `true` and this will be displayed correctly in `help` output.

Reply:
Done